rich_pixels/_pixel.py

```diff
@@ -26,30 +27,30 @@ def __init__(self) -> None:
 
     @staticmethod
     def from_image(
-        image: "Image",
+        image: Image,
     ):
         segments = Pixels._segments_from_image(image)
         return Pixels.from_segments(segments)
 
     @staticmethod
     def from_image_path(
-        path: PurePath | str,
-        resize: tuple[int, int] | None = None,
+        path: Union[PurePath, str],
+        resize: Optional[Tuple[int, int]] = None,
     ) -> Pixels:
         """Create a Pixels object from an image. Requires 'image' extra dependencies.
 
         Args:
             path: The path to the image file.
             resize: A tuple of (width, height) to resize the image to.
         """
-        with Image.open(Path(path)) as image:
+        with PIL.Image.open(Path(path)) as image:
             segments = Pixels._segments_from_image(image, resize)
 
         return Pixels.from_segments(segments)
 
     @staticmethod
     def _segments_from_image(
-        image: "Image", resize: tuple[int, int] | None = None
```

Comment:
The reason I used "Image" here was because I think it avoids a crash when the `image` extras aren't installed.

Reply:
Actually, I think at the moment things  won't work without the `image` extras. I think I will remove them as extras and just make them a requirement anyway.